App/Program.cs

```
@@ -209,36 +209,111 @@ public static bool ExecuteFolder(Config config)
+                        if (config.StackTraces) Console.Error.WriteLine(ex.ToString());
+                        else Console.Error.WriteLine(ex.Message);
+                        
+                        migrationSuccess = false;
+                        break;
+                    }
+                    Console.WriteLine("{0} - {1}", result, file);
+                    if (!config.Force && result == MigrationResult.Changed)
+                    {
+                        migrationSuccess = false;
+                        break;
+                    }
+                    if (config.Verbose) Console.WriteLine("Info: ...done!");
                 }
-                if (config.Verbose) Console.WriteLine("Info: ...done!");
+
+                // Restart TimescaleDB background workers if they were stopped
+                if (timescaleBackgroundWorkersStopped)
+                {
+                    try
+                    {
+                        if (config.Verbose)
+                        {
+                            Console.WriteLine("Info: Starting TimescaleDB background workers...");
+                        }
+
+                        connection.Execute("SELECT _timescaledb_functions.start_background_workers();");
+                        
+                        if (config.Verbose)
+                        {
+                            Console.WriteLine("Info: TimescaleDB background workers started successfully");
+                        }
+                    }
+                    catch (Exception ex)
+                    {
+                        Console.WriteLine($"Warning: Failed to restart TimescaleDB background workers: {ex.Message}");
+                        if (!migrationSuccess)
+                        {
+                            Console.WriteLine("Warning: You may need to manually restart TimescaleDB background workers using: SELECT _timescaledb_functions.start_background_workers();");
```

I would return false in this case. If it's just a line in the log, people will miss it.

---

Doesn't look like this is resolved correctly @jonatas.

We need to either:
- `return false`
- set `migrationSuccess = false` 

In order for the application to break if `SELECT _timescaledb_functions.start_background_workers();` throws, no?

---

ok! makes sense! sorry I missed addressing it and thought I already did 🫠 
Agreed to make it fail early. The only point was a not having a way to proceed or force execution if the command fails for another reason but I can't imagine it happening too.

---

I pushed the version. One detail I kept the `--force` option to continue, let me know if it's not a good idea.